newswires/client/src/icons.ts

```diff
@@ -28,10 +31,12 @@ import { icon as popout } from '@elastic/eui/es/components/icon/assets/popout';
 import { icon as refresh } from '@elastic/eui/es/components/icon/assets/refresh';
 import { icon as returnKey } from '@elastic/eui/es/components/icon/assets/return_key';
 import { icon as search } from '@elastic/eui/es/components/icon/assets/search';
+import { icon as sortRight } from '@elastic/eui/es/components/icon/assets/sortRight';
```

Comment:
I think we need to add `sortLeft`, too. When I select the 'Absolute' time option I get the following:

`Uncaught Error: Module not found in bundle: ./assets/sortLeft`